src/index.js

```diff
@@ -308,7 +313,7 @@ class IpfsRepo {
   async _storageMaxStat () {
     try {
       const max = await this.config.get('Datastore.StorageMax')
-      return new Big(max)
+      return new Big(bytes(max))
```

Comment:
Why is this necessary?

Reply:
This is necessary because `datastore` config can have a default value as you can see [here](https://github.com/ipfs/js-ipfs-repo/blob/master/test/test-repo/config#L9) and this need to be parsed to bytes. Otherwise, `Big` object won't be properly instantiated.